jme3-android/src/main/java/com/jme3/system/android/JmeAndroidSystem.java

```diff
@@ -35,6 +37,18 @@ public class JmeAndroidSystem extends JmeSystemDelegate {
         } catch (UnsatisfiedLinkError e) {
         }
     }
+
+    public JmeAndroidSystem(){
```

Comment:
Shouldn't there be a settings handler also provided here (that does nothing and only returns true like the old behavior)?